src/pages-conditional/developers/local-environment.js

```diff
@@ -274,6 +274,7 @@ const frameworksList = [
     name: "Waffle",
     description:
       "The most advanced testing lib for smart contracts. Use alone or with Scafold-eth or Hardhat.",
+    gitHubUrl: "https://github.com/EthWorks/waffle",
```

Comment:
Since we're already using the GitHub API to pull in star counts, we could also use the API to pull in GitHub URL. See example code in my comment below.